x-pack/plugins/observability/public/components/shared/page_template/page_template.tsx

```diff
@@ -41,6 +44,7 @@ export interface ObservabilityPageTemplateDependencies {
   getUrlForApp: ApplicationStart['getUrlForApp'];
   navigateToApp: ApplicationStart['navigateToApp'];
   navigationSections$: Observable<NavigationSection[]>;
+  Context?: FC;
```

Comment:
React's `ComponentType` (defined as `type ComponentType<P = {}> = ComponentClass<P> | FunctionComponent<P>;`) is probably better to use here since it will allow any class or function component.